app/controllers/products_controller.rb

```diff
@@ -26,7 +28,7 @@ def create
   def update
     respond_to do |format|
       @product.update(product_params) ? format.html { redirect_to @product, notice: 'Product updated successfully!' }
```

Comment:
if the if else clause is too complex (does not fit in one line or it's hard to read), it's not recommended to use the short format, I would replace this with a normal `if else` on multiple lines - readability is always top 1 priority :-)